modules/uikit/CMakeLists.txt

```diff
@@ -22,14 +25,17 @@ set(${PROJECT_NAME}_incPaths
     "../../thirdparty/next/inc"
     "../../thirdparty/next/inc/math"
     "../../thirdparty/next/inc/core"
-	"../../thirdparty/pugixml/src"
-	"../../thirdparty/cssparser/include"
+    "../../thirdparty/pugixml/src"
+    "../../thirdparty/cssparser/include"
 )
 
 # This path is only needed on the BSDs
 if(UNIX AND NOT APPLE AND NOT LINUX)
     set(${PROJECT_NAME}_incPaths
         ${${PROJECT_NAME}_incPaths}
+        include_directories(${Qt5Gamepad_INCLUDE_DIRS})
```

Comment:
I'm not sure Qt5Gamepad is required here it used only in WorldEditor application and will be removed soon

Reply:
Removing now. There may be some lingering code in uikit or a Qt header that needed it to work. Won't be able to look into it much until after work, though.